code/__DEFINES/subsystems.dm

```diff
@@ -142,34 +142,35 @@
 // Subsystem fire priority, from lowest to highest priority
 // If the subsystem isn't listed here it's either DEFAULT or PROCESS (if it's a processing subsystem child)
 
-#define FIRE_PRIORITY_PING			10
-#define FIRE_PRIORITY_IDLE_NPC		10
-#define FIRE_PRIORITY_SERVER_MAINT	10
-#define FIRE_PRIORITY_RESEARCH		10
-#define FIRE_PRIORITY_VIS			10
-#define FIRE_PRIORITY_GARBAGE		15
-#define FIRE_PRIORITY_WET_FLOORS	20
-#define FIRE_PRIORITY_AIR			20
-#define FIRE_PRIORITY_NPC			20
-#define FIRE_PRIORITY_PROCESS		25
-#define FIRE_PRIORITY_THROWING		25
-#define FIRE_PRIORITY_SPACEDRIFT	30
-#define FIRE_PRIORITY_FIELDS		30
-#define FIRE_PRIOTITY_SMOOTHING		35
-#define FIRE_PRIORITY_NETWORKS		40
-#define FIRE_PRIORITY_OBJ			40
-#define FIRE_PRIORITY_ACID			40
-#define FIRE_PRIOTITY_BURNING		40
-#define FIRE_PRIORITY_DEFAULT		50
-#define FIRE_PRIORITY_PARALLAX		65
-#define FIRE_PRIORITY_MOBS			100
-#define FIRE_PRIORITY_TGUI			110
-#define FIRE_PRIORITY_TICKER		200
-#define FIRE_PRIORITY_ATMOS_ADJACENCY	300
-#define FIRE_PRIORITY_CHAT			400
-#define FIRE_PRIORITY_OVERLAYS		500
-#define FIRE_PRIORITY_EXPLOSIONS	666
-#define FIRE_PRIORITY_INPUT			1000 // This must always always be the max highest priority. Player input must never be lost.
+#define FIRE_PRIORITY_PING				10
```

Comment:
@MrStonedOne is it valid to have subsystems with same priority? seems self defeating?

Reply:
Priorities are a weight, not just a sort index.